cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```
@@ -128,6 +128,19 @@ spec:
       #     volumeMode: Block
       #     accessModes:
       #       - ReadWriteOnce
+      # dedicated block device to store bluestore wal (block.wal)
+      # - metadata:
+      #     name: waldata
+      #   spec:
+      #     resources:
+      #       requests:
+      #         # Find the right size https://docs.ceph.com/docs/master/rados/configuration/bluestore-config-ref/#sizing
+      #         storage: 5Gi
+      #     # IMPORTANT: Change the storage class depending on your environment (e.g. local-storage, gp2)
+      #     storageClassName: gp2
```

io1 to match the documentation?
```suggestion
      #     storageClassName: io1
```